.github/workflows/build.yml

```diff
@@ -89,6 +89,10 @@ jobs:
       - name: Test publish to local
         run: ./gradlew publishToMavenLocal -x test -PjdkVersion=${{ matrix.java-version }}
 
+      - name: Free up disk space
+        run: |
+          dev/ci/util_free_space.sh
```

Comment:
The reason you added this script is that the failed case you pasted has run out of disk space. Am I right?

Reply:
Yes. 
```
Disk usage:
Filesystem      Size  Used Avail Use% Mounted on
overlay          73G   68G  4.9G  94% /
```
```
ERROR: Doris FE (version < 2.1.0) can not start with less than 6G disk space.
```

Reply:
I see.